docs/CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [4.3.2] - 2024-09-19
 ### Changed
 - Update the release documentation (OSIDB-3384)
+- Tracker validations show affect's module/component (OSIDB-3439)
```

Comment:
Wrong ## section

Reply:
nice catch, I didn't paid enough attention in the rebase.